packages/angular_devkit/build_angular/src/webpack/plugins/javascript-optimizer-worker.ts

```diff
@@ -93,7 +93,7 @@ async function optimizeWithTerser(
     { [name]: code },
     {
       compress: {
-        passes: advanced ? 3 : 1,
+        passes: advanced ? 2 : 1,
```

Comment:
If I recall correctly 3 yielded better results when using `pure_getters`. Isn't this the case any more?

Reply:
Added some project comparisons in the PR summary.  The change is quite small (> 1%) and the third pass also increases the build time.